functions/vercel/api/delete/movies.ts

```diff
@@ -0,0 +1,29 @@
+import 'isomorphic-fetch'
+import type { VercelRequest, VercelResponse } from '@vercel/node'
+// import { getXataClient } from '../../_lib/xata.codegen'
+
+export default async function handler(
+  request: VercelRequest,
+  response: VercelResponse
+) {
+  // const xata = getXataClient()
+  // const { id } = request.body
+
+  if (request.method !== 'DELETE') {
+    response.status(405).end('Only `DELETE` requests')
```

Comment:
A `return` is missing here and in a few other 405 responses.

Maybe it'd be useful to do something like https://github.com/xataio/frontend-next/blob/main/utils/errors.ts or just copy and paste that.

What we do in the frontend to avoid missing things like that is throwing an exception. Then we have a function that wraps all handlers to catch the exception, and depending on its type it sends the corresponding status code. It's also useful to make all errors consistent and return always JSON with the same structure.

Reply:
Thanks. That's a good point.

I've been trying to avoid abstractions (as we talked, I removed the dynamic routes).

But an error handler can be useful even for contrived use-cases like this one.
I'll refactor 👍